src/changelogs/tags.py

```diff
@@ -275,3 +274,102 @@ def __init__(self, default_tag):
             'HeroLab Magician': ['Magician', 'Hero'],
             'HeroLab Viper': ['Viper', 'Hero'],
         }
```

Comment:
It seems like we have two places that define tag relationships. Shouldn't a simple tag tree, as you have below, cover that fully?

Reply:
Correct, newly added self.tag_tree is going to replace self.parents. I haven't finished quite yet so I haven't removed the old one, still a few things to look over.